proposals/NNNN-GetInteriorVehicleData.md

```diff
@@ -0,0 +1,96 @@
+# Interior Vehicle Data Subscription Management and Data Caching in SDL
+
+* Proposal: [SDL-NNNN](NNNN-GetInteriorVehicleData.md)
+* Author: [Zhimin Yang](https://github.com/smartdevicelink/yang1070)
+* Impacted Platforms: [Core / Web / RPC ]
+
+## Introduction
+
+This proposal is for improving the process of GetInteriorVehicleData RPC in SDL core. This proposal can reduce the number of messages between SDL and HMI, and improves the response time of GetInteriorVehicleData a lot. It accomplishes this by introducing a cache of the interior vehicle data and maintains SDL's subscription status of a module(Type). 
+
+
+## Motivation
+
+The following sequence diagram shows the current behavior of sdl regarding GetInteriorVehicleData.
+
+![../assets/proposals/nnnn-getInteriorVehicleData/GetInteriorVehicleData_current.png](../assets/proposals/nnnn-getInteriorVehicleData/GetInteriorVehicleData_current.png)
+
+#### 1.	SDL always forwards a valid GetInteriorVehicleData request from any app to HMI, no matter parameter `subscribe` included or not.
+#### 2. SDL expects HMI to manage each app’s subscription status (per moduleType) and maintains a local status keep tracking of each application’s subscription (per moduleType).
```

Comment:
Please update to "...local status to keep track of each..."

Reply:
changed